tools/packaging/guest-image/build_image.sh

```diff
@@ -26,13 +26,23 @@ build_initrd() {
 	info "Build initrd"
 	info "initrd os: $initrd_distro"
 	info "initrd os version: $initrd_os_version"
-	sudo -E PATH="$PATH" make initrd \
+	rootfs_build_dest="${builddir}/initrd-image"
+	makeopts='
 		DISTRO="$initrd_distro" \
 		DEBUG="${DEBUG:-}" \
 		OS_VERSION="${initrd_os_version}" \
-		ROOTFS_BUILD_DEST="${builddir}/initrd-image" \
+		ROOTFS_BUILD_DEST="${rootfs_build_dest}" \
 		USE_DOCKER=1 \
-		AGENT_INIT="yes"
+		AGENT_INIT="yes" \
+		SKOPEO_UMOCI="${SKOPEO_UMOCI:-}" \
+		AA_KBC="${AA_KBC:-}"
+
+	'
+	eval sudo -E PATH="$PATH" make rootfs $makeopts
+	if [ -n "${INCLUDE_ETC:-}" ]; then
+		sudo cp ${INCLUDE_ETC} "${rootfs_build_dest}/${initrd_distro}_rootfs/etc/"
```

Comment:
Are perms and ownership important (aka do we need to use `install(1)` here)?

Reply:
Good call; yes, we do. I can't say I'm very happy with this solution (you can specify multiple space-separated files), which is the main reason this is a draft. Would you know of a more elegant (low-code :)) solution? Maybe even just using a folder is nicer.

Reply:
`cp -aL` might be sufficient then?

Reply:
Hm, on second thought I think we should preserve mode (no use cases known to me right now, but who knows), but ownership should be `root` (as everything in that rootfs), i.e. `sudo cp -LR --preserve=mode ${INCLUDE_ETC}/* …`. Limiting to `/etc` might be too inflexible, I'm growing more fond of having a "merge rootfs" instead, so you could have a folder with `etc`, `var`, `root`, whatever and that would just get merged.